go/vt/vtctl/workflow/mount.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package workflow
+
+import (
+	"context"
+
+	"vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/vterrors"
+
+	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
+	vtrpcpb "vitess.io/vitess/go/vt/proto/vtrpc"
+)
+
+func notExistsError(name string) error {
+	return vterrors.New(vtrpcpb.Code_INVALID_ARGUMENT, "there is no vitess cluster named "+name)
+}
+
+func (s *Server) MountRegister(ctx context.Context, req *vtctldatapb.MountRegisterRequest) (*vtctldatapb.MountRegisterResponse, error) {
+	vci, err := s.ts.GetExternalVitessCluster(ctx, req.Name)
+	if err != nil {
+		return &vtctldatapb.MountRegisterResponse{}, vterrors.Wrap(err, "failed to get external vitess cluster in MountRegister")
```

Comment:
It will be easier to read this way, I think:
```
vterrors.Errorf(vtrpcpb.Code_INTERNAL, "failed to get external vitess cluster in MountRegister: %v", err)
```

Same for the other cases below as wrapping will make it seem like there's an inner and outer error but here there's only one error that we're adding context to.